dev-howitz.toml

```diff
@@ -5,50 +5,53 @@ DEBUG = true
 [howitz]
 storage = "./howitz.sqlite3"
 devmode = true
-poll_interval = 30
+poll_interval = 60
 timezone='LOCAL'
 
 [zino.connections.default]
 server =
 
-[zino.connections.other]
-server =
+## Comment out and fill out in order to configure another Zino server profile
+#[zino.connections.other]
+#server =
 
 [logging]
 version = 1
 
 [logging.root]
-level = "DEBUG"
-
-[logging.formatters.default]
-format = "%(levelname)s %(name)s in %(funcName)s: %(message)s"
-
-[logging.formatters.detail]
-format = "%(asctime)s %(levelname)s %(name)s %(pathname)s:%(lineno)s:: %(message)s"
-
-[logging.handlers.null]
-class = "logging.NullHandler"
-
-[logging.handlers.wsgi]
-class = "logging.StreamHandler"
-stream = "ext://flask.logging.wsgi_errors_stream"
-formatter = "default"
-
-[logging.handlers.error]
-level = "WARNING"
-class = "logging.FileHandler"
-filename = "errors.log"
-formatter = "detail"
-
-[logging.handlers.debug]
-level = "DEBUG"
-class = "logging.handlers.TimedRotatingFileHandler"
-filename = "debug.log"
-when = 'D'
-formatter = "detail"
-
-[logging.loggers.zinolib]
-handlers = ["debug", "wsgi"]
-
-[logging.loggers.howitz]
-handlers = ["debug", "wsgi"]
+level = "ERROR"
+
+## Comment out and tweak in order to configure logging level for dev or prod
+#[logging.formatters.default]
+#format = "%(levelname)s %(name)s in %(funcName)s: %(message)s"
+#
```

Comment:
With this, the config file is suitable neither for development nor production without major alterations. I'm not sure this is a good idea. @lunkwill42?

Reply:
If we want a clean separation, the example config file referenced in the README should not be a file called `dev-howitz.toml`.  `howitz.toml.example` could do - then we could keep a `howitz.dev.toml.example` or something.  Normal folks do not want to start with the latter file.

Reply:
If we want a clean separation, the example config file referenced in the README should not be a file called `dev-howitz.toml`.  `howitz.toml.example` could do - then we could keep a `howitz.dev.toml.example` or something.  Normal folks do not want to start with the latter file, @hmpf @podliashanyk